Project.toml

```diff
@@ -16,11 +16,11 @@ SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 
 [compat]
 ArnoldiMethod = "0.0.4"
-ColorTypes = "0.9"
-Colors = "0.11"
+ColorTypes = "0.9, 0.10"
+Colors = "0.11, 0.12"
 Compose = "0.7, 0.8"
-LightGraphs = "1.1"
-VisualRegressionTests = ">= 0.2"
+LightGraphs = "1.1, 1.2, 1.3"
```

Comment:
this one should not be necessary? All 1.x versions should be fine greater that 1.1

Reply:
ah - this is a difference between pre and post 0.x release - right?

Reply:
yes exactly, otherwise this looks good, merging when it goes green